frontend/src/i18n/locales/en.yaml

```diff
@@ -163,6 +163,18 @@ series:
     text: >
       The data of the requested series has not been fully transferred yet. This should
       happen automatically soon. Try again in a few minutes.
+  settings:
+    order: Order
+    order-label: Choose video order
+    old-to-new: Oldest first
```

Comment:
`new-to-old` mysteriously disappeared (this is also obvious when checking test deployment in English)